### PR TITLE
chore: add wmg snapshot example queries

### PR DESCRIPTION
**Pull request by atolopko-czi**
- #N/A

### Reviewers
**Functional:** 
@Bento007 

**Readability:** 

---

## Changes
add wmg snapshot example queries. for inspection, troubleshooting, and pedagogical purposes.

noted in docs [here](https://docs.google.com/document/d/1xgRVUIP9tfbYAFeGR5KmRnT31P6JpV5VxpihbXX_eDY/edit#heading=h.u0bfvvuobfnt)

### CODE REVIEW

File: backend/scripts/wmg_query_examples.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from backend.wmg.data.snapshot import _open_cube
+
+os.environ['DEPLOYMENT_STAGE'] = 'dev'
+
+pd.set_option('max_columns', 10)
+pd.set_option('display.width', 256)
+
+
+def load_snapshot(snapshot_identifier) -> WmgSnapshot:
+    snapshot_base_uri = f"s3://cellxgene-wmg-dev/{snapshot_identifier}/"
```
Comment (atarashansky) [inline backend/scripts/wmg_query_examples.py:21]: Should it be instead:

`snapshot_base_uri = f"s3://cellxgene-wmg-{os.environ['DEPLOYMENT_STAGE']}/{snapshot_identifier}/"`

File: backend/scripts/wmg_query_examples.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    # tissues = ["UBERON:0002368"]
+    organism = "NCBITaxon:9606"
+    # print(q._snapshot.expression_summary_cube.multi_index[genes, :, :, organism])
+    #
+    # print(q._snapshot.expression_summary_cube[genes[0], tissues[0], :, organism])
+    # print(q._snapshot.expression_summary_cube.multi_index[genes, tissues, [], organism])
+    print(q._snapshot.expression_summary_cube.query().multi_index[genes, tissues, [], organism])
+    print(pd.DataFrame(q._snapshot.expression_summary_cube.multi_index[genes, tissues, [], organism]))
+    print(
+        q._snapshot.expression_summary_cube.query(use_arrow=False, attrs=["n_cells"]).df[genes, tissues, :, organism].shape
```
Comment (atarashansky) [inline backend/scripts/wmg_query_examples.py:50]: Add comment about why `use_arrow=False` here and not in previous lines?

**Review by atarashansky — APPROVED** [review]
Looks good, couple minor comments. Approving as is. Thanks for making this!!

File: backend/scripts/wmg_query_tiledb_api_examples.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+from backend.wmg.data.snapshot import _open_cube, EXPRESSION_SUMMARY_CUBE_NAME
+
+pd.set_option('max_columns', 10)
+pd.set_option('display.width', 256)
+
+os.environ['DEPLOYMENT_STAGE'] = 'dev'
+
+if __name__ == '__main__':
+    cube = _open_cube(f's3://cellxgene-wmg-dev/1662103227/{EXPRESSION_SUMMARY_CUBE_NAME}/')
```
Comment (Bento007) [inline backend/scripts/wmg_query_tiledb_api_examples.py:15]: as @atarashansky mentioned above, make the bucket name dynamic based on the DEPLOYMENT_STAGE

**Review by Bento007 — APPROVED** [review]

File: backend/scripts/wmg_query_examples.py (hunk 2 above)
Comment (atolopko-czi) [reply backend/scripts/wmg_query_examples.py:50]: removed, was a tiledb bug workaround

**Pull request merged**